src/__tests__/element-queries.js

```diff
@@ -551,12 +551,14 @@ test('query/get element by its title', () => {
         <span title="Ignore this" id="1"/>
         <span title="Delete" id="2"/>
         <span title="Ignore this as well" id="3"/>
+        <div title="WrongTitle" id="4">HelloWorld</div>
     </div>
   `)
 
   expect(getByTitle('Delete').id).toEqual('2')
   expect(queryByTitle('Delete').id).toEqual('2')
   expect(queryByTitle('Del', {exact: false}).id).toEqual('2')
+  expect(queryByTitle("HelloWorld")?.id).not.toEqual('4')
```

Comment:
Let's avoid conditionals in tests. We're assuming that this will return `null` right? So should this be:

```suggestion
  expect(queryByTitle("HelloWorld")).toBeNull()
```

Reply:
Yup, that woorks too!